maquetado/index.html

```diff
@@ -11,21 +11,28 @@
 </head>
 
 <body>
-  <div class="book-detail">
-    <p class="book-title">Titulo del Libro</p>
-    <p class="book-description book-genre">(género)</p>
-  </div>
-  <div class="book-detail">
-    <p class="book-titleDescription">Autor del libro:</p>
-    <p class="book-description">Nombre del autor del libro</p>
-  </div>
-  <div class="book-detail">
-    <h1 class="book-titleDescription">Editorial:</h1>
-    <p class="book-description">Nombre de la editorial</p>
-  </div>
-  <div class="book-detail">
-    <p class="book-titleDescription">Año de publicación:</p>
-    <p class="book-description">Año de publicación</p>
+  <div class="book-content">
+    <img class="image" src="image/book-cover.png" alt="Book Image">
+    <div>
+      <div class="book-detail">
+        <p class="book-title">Titulo del Libro</p>
+        <p class="book-description book-genre">(género)</p>
+      </div>
+      <div class="book-separator">
+      </div>
```

Comment:
You don't need to set a new div for this. You can use pseudo-classes ::after or ::before to make the green line separator. The first book detail div can have a different name so you can point to it specifically and apply the pseudo class.